src/pages/Research/Content/ResearchDescription.tsx

```diff
@@ -120,7 +120,7 @@ const ResearchDescription = ({ research, isEditable, ...props }: IProps) => {
             </AuthWrapper>
           ) : null}
           {/* Check if research should be moderated */}
-          {props.needsModeration && (
+          {props.needsModeration && research.moderation !== 'draft' && (
```

Comment:
```suggestion
          {props.needsModeration && research.moderation === 'awaiting-moderation' && (
```

This is the logic we would want